### DISCUSSION TITLE
Question: How to generate msvc project files with progen?

### DISCUSSION THREAD

**Discussion by JackyWoo** [pocoproject/poco#3746] [Q&A] [answered]
How to generate msvc project files with progen? Can it be generated on MacOS?

**Comment by aleks-f** [comment, marked as answer]
No, Progen requires Windows and Visual Studio 2017, 2019 or 2022. It uses VS_90 project as a template for generation of the target project.

**Reply by JackyWoo** [reply]
docs :  [Generating-VS-Projects-using-Progen](https://github.com/pocoproject/poco/wiki/Generating-VS-Projects-using-Progen)
